gui/browsable/src/RNTupleBrowseProvider.cxx

```diff
@@ -152,14 +152,14 @@ class RNTupleElement : public RElement {
 
 class RFieldsIterator : public RLevelIter {
 
-   std::shared_ptr<ROOT::Experimental::RNTupleReader> fNtplReader;
+   std::shared_ptr<ROOT::RNTupleReader> fNtplReader;
    std::vector<ROOT::DescriptorId_t> fFieldIds;
    std::string fParentName;
    int fCounter{-1};
 
 public:
-   RFieldsIterator(std::shared_ptr<ROOT::Experimental::RNTupleReader> ntplReader,
-                   std::vector<ROOT::DescriptorId_t> &&ids, const std::string &parent_name = ""s)
+   RFieldsIterator(std::shared_ptr<ROOT::RNTupleReader> ntplReader, std::vector<ROOT::DescriptorId_t> &&ids,
+                   const std::string &parent_name = "" s)
```

Comment:
```suggestion
                   const std::string &parent_name = ""s)
```
This is causing the builds to fail

Reply:
Is it? I see the build failing for cmake errors...
That space was added by clang-format and it's not failing to me locally, so I need to double-check with gcc...

Reply:
It was failing on macOS 14, so could be an Apple clang thing

Reply:
I tried locally and it actually fails (I didn't notice that cmake had silently disabled my webgui=on)...I just removed the `s` since it's not necessary.